src/wallet/wallet2.cpp

```diff
@@ -13262,9 +13262,8 @@ crypto::public_key wallet2::get_tx_pub_key_from_received_outs(const tools::walle
   return tx_pub_key;
 }
 
-bool wallet2::export_key_images(const std::string &filename, bool all) const
+std::string wallet2::export_key_images_string(bool all) const
```

Comment:
From how I understand things at this time, I fully support to add this function.

For those who like to read, here is a snippet from the notes I took while working on the wallet API:
>I am wondering, for the `wallet2::export_key_images()` ([src](https://github.com/monero-project/monero/blob/a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623/src/wallet/wallet2.cpp#L13295-L13346)) that's used by the wallet-rpc ([here](https://github.com/monero-project/monero/blob/a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623/src/wallet/wallet_rpc_server.cpp#L3037)), afaiui it sends the key images and signatures in clear text, wouldn't it be safer to change it, so it returns a string of the encrypted key images and signatures?
If my concern is valid, I would not add the `export_key_images()` function mentioned above to the API, but propose to add another `std::string export_key_images(bool all)` in wallet2 and add that to the API instead, it would return an encrypted string of key images and signatures as created [here](https://github.com/monero-project/monero/blob/a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623/src/wallet/wallet2.cpp#L13273-L13291) (`magic + ciphertext`).
That should be safer to use for the wallet-rpc compared to what it's using now and I assume it would be the most versatile for every client using the wallet API.